docker-images/kafka/cruise-control-scripts/cruise_control_config_generator.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+CC_CAPACITY_FILE="/tmp/capacity.json"
+CC_CLUSTER_CONFIG_FILE="/tmp/clusterConfig.json"
+
+# Generate capacity file
+# TODO: Update DISK value based on volume sizes
+cat <<EOF > $CC_CAPACITY_FILE
+{
+	"brokerCapacities": [{
+		"brokerId": "-1",
+		"capacity": {
+			"DISK": "100000",
+			"CPU": "100",
+			"NW_IN": "10000",
+			"NW_OUT": "10000"
+		},
+		"doc": "This is the default capacity. Capacity unit used for disk is in MB, cpu is in percentage, network throughput is in KB."
+	}]
+}
+EOF
+
+# Generate cluster config
+# TODO: Add Kafka configurations here like "min.insync.replicas" for CC reporting
```

Comment:
Do you know how you're going to do this?

Reply:
I was thinking of just copying the whole Kafka configuration here from KAFKA_CONFIGURATION env var or , if possible, reference the used Kafka configuration file, `/tmp/strimzi.properties` from this file. Right now, only `min.insync.replicas` setting is used by CC from this file